<launch>
	<arg name="nav_use_rotvel" default="true"/>
	<arg name="map" default="map"/>
	<node name="map_server" pkg="map_server" type="map_server" args="$(find yahboomcar_nav)/maps/$(arg map).yaml"/>

	<include file="$(find yahboomcar_nav)/launch/library/amcl.launch"/>
	<include file="$(find yahboomcar_nav)/launch/library/move_base.launch">
		<arg name="nav_use_rotvel" value="$(arg nav_use_rotvel)"/>
	</include>

  <node name="snake_explorer" pkg="snake_explorer" type="snake_explorer" output="screen">
    <param name="min_x" value="-1000.0"/>
    <param name="max_x" value="1000.0"/>
    <param name="min_y" value="-1.0"/>
    <param name="max_y" value="1000.0"/>
  </node>
</launch>
